pkgs/applications/qubes/qubes-core-vchan-xen/default.nix

```diff
@@ -28,8 +28,9 @@ stdenv.mkDerivation rec {
     description = "Libraries required for the higher-level Qubes daemons and tools";
     homepage = "https://qubes-os.org";
     license = licenses.gpl2Plus;
-    maintainers = with maintainers; [ _0x4A6F ];
+    maintainers = with maintainers; [ ];
     platforms = platforms.linux;
+    broken = true;
```

Comment:
Similarly, to libvmi, you want https://github.com/QubesOS/qubes-core-vchan-xen/commit/8c4c3807119f27957e6c7f87d505d66d0ea4c3d0

Reply:
For Qubes, I'm planning to maintain all of the Qubes utilities, including their customised Xen. Similarly, this is temporarily set as broken until this update is merged and I get to work on the Qubes packages.

Reply:
> For Qubes, I'm planning to maintain all of the Qubes utilities, including their customised Xen. -- @SigmaSquadron (https://github.com/NixOS/nixpkgs/pull/324693#discussion_r1685664505)

Are there any utilities for Xen that use GTK? It's what NiXium uses as a stable frontend environment.

Reply:
Please keep PR reviews vaguely relevant instead of pinging everyone subscribed to ask an off‐topic question and promote your project again…